runtime/calamari/tests/integration_tests.rs

```diff
@@ -532,189 +530,64 @@ fn root_can_change_default_xcm_vers() {
-            (bob.clone(), INITIAL_BALANCE),
-            (charlie.clone(), INITIAL_BALANCE),
-        ])
-        .with_collators(vec![alice.clone()], desired_candidates)
-        .build()
-        .execute_with(|| {
-            assert_ok!(Session::set_keys(
-                Origin::signed(bob.clone()),
-                bob_session_keys.clone(),
-                vec![]
-            ));
-
-            assert_ok!(Session::set_keys(
-                Origin::signed(charlie.clone()),
-                charlie_session_keys.clone(),
-                vec![]
-            ));
-
-            assert_eq!(CollatorSelection::candidates(), vec![]);
-
-            assert_ok!(Utility::batch(
-                root_origin(),
-                vec![
-                    Call::CollatorSelection(manta_collator_selection::Call::register_candidate {
-                        new_candidate: bob.clone(),
-                    }),
-                    Call::CollatorSelection(manta_collator_selection::Call::register_candidate {
-                        new_candidate: charlie.clone(),
-                    }),
-                ],
-            ));
-
-            let candidate_bob = manta_collator_selection::CandidateInfo {
-                who: bob.clone(),
-                deposit: BOND_AMOUNT,
-            };
-            let candidate_charlie = manta_collator_selection::CandidateInfo {
-                who: charlie.clone(),
-                deposit: BOND_AMOUNT,
-            };
-            assert_eq!(
-                CollatorSelection::candidates(),
-                vec![candidate_bob, candidate_charlie]
-            );
-            assert_eq!(
-                Session::queued_keys(),
-                vec![(alice.clone(), alice_session_keys.clone()),]
-            );
-            assert_eq!(Session::validators(), vec![alice.clone()]);
-
-            let mut session_index = 1;
-            run_to_block(session_index * Period::get());
-
-            let all_collator_pairs = vec![
-                (alice.clone(), alice_session_keys.clone()),
-                (bob.clone(), bob_session_keys.clone()),
-                (charlie.clone(), charlie_session_keys.clone()),
-            ];
-
-            assert_eq!(Session::queued_keys(), all_collator_pairs);
-            assert_eq!(Session::validators(), vec![alice.clone()]);
-
-            session_index += 1;
-            run_to_block(session_index * Period::get());
-
-            assert_eq!(Session::queued_keys(), all_collator_pairs);
-            assert_eq!(
-                Session::validators(),
-                vec![alice.clone(), bob.clone(), charlie.clone()]
-            );
-        });
-}
+// TODO: Full integration test for staking
+// fn advance_session_assertions(session_index: &mut u32, advance_by: u32) {
+//     *session_index += advance_by;
+
+//     run_to_block(*session_index * Period::get() - 1);
+//     assert_eq!(Session::session_index(), *session_index - 1);
+
+//     run_to_block(*session_index * Period::get());
```

Comment:
forgotten commented code

Reply:
Postponed to https://github.com/Manta-Network/Manta/issues/798